libpod/container.go

```diff
@@ -123,6 +123,24 @@ type ContainerConfig struct {
 	// TODO allow overriding of log path
 }
 
+func (t ContainerState) String() string {
+	switch t {
+	case ContainerStateUnknown:
```

Comment:
As noted above, I think the status ought to be defined in some common area and should be using a data structure like an enum or some such to avoid mismatches between strings.  To many past burn marks to list.